client/allocrunnerv2/alloc_runner.go

```diff
@@ -90,7 +92,7 @@ func NewAllocRunner(config *Config) (*allocRunner, error) {
 		vaultClient:  config.Vault,
 		tasks:        make(map[string]*taskrunner.TaskRunner, len(tg.Tasks)),
 		waitCh:       make(chan struct{}),
-		updateCh:     make(chan *structs.Allocation),
+		updateCh:     make(chan *structs.Allocation, 1), // see comment above
```

Comment:
Should this be a named variable instead, with the comment to explain why it is set to 1?